back-end/src/main/java/com/PhanLam/backend/model/Lesson.java

```diff
@@ -82,10 +76,19 @@
     @Column (name = "DateCreated", nullable = false)
     @Temporal (TemporalType.TIMESTAMP)
     private Date dateCreated;
+    @Column (name = "LastModified")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date lastModified;
+    private static final long serialVersionUID = 1L;
+    @Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
+    @Basic (optional = false)
+    @Column (name = "LessonID", nullable = false)
+    private Integer lessonID;
     @OneToMany (mappedBy = "lessonID", fetch = FetchType.LAZY)
     private List<Document> documentList;
     @JoinColumn (name = "CourseID", referencedColumnName = "CourseID", nullable = false)
-    @ManyToOne (optional = false, fetch = FetchType.LAZY)
+    @ManyToOne (optional = false, fetch = FetchType.EAGER)
```

Comment:
@BuiXuanNhanTriSE05211 Ông giữ mỗi thay đổi của dòng này thôi (Lazy -> Eager). Rồi làm tương tự như các file trên: 
-> Ông xóa hết đi rồi copy toàn bộ nội dung file này của tôi trong nhánh develop paste vào nhé.

Reply:
Nghĩa là giữ lại dòng số 91:  "@ManyToOne (optional = false, fetch = FetchType.EAGER)" còn lại xóa hết copy lại à. Nếu xóa hết cả file thì dòng này để lại sẽ để ở chỗ nào? Ô giải thích lại đi, đang chưa hiểu rõ lắm.